source/_docs/authentication/multi-factor-auth.markdown

```diff
@@ -75,15 +75,17 @@ Add Notify MFA to your `configuration.yaml` file like this:
 homeassistant:
   auth_mfa_modules:
     - type: notify
+      include:
```

Comment:
If I remember correctly we want to keep the example code very basic without optional config keys.

Reply:
It wasn't clear for me that you have to define only the name of the notify component. Thats a little off to the regular Home Assistant config.